src/components/Game/Game.tsx

```diff
@@ -95,7 +91,7 @@ export function Game() {
   }
 
   return (
-    <div>
+    <>
```

Comment:
ezt nem értem

Reply:
Summary: that `div` had no effect other than wrapping the child elements, so to avoid nesting a bunch of `div`s, we can just wrap them in a `<>` so the DOM doesn’t get flooded.